src/styles/styles.js

```diff
@@ -1017,6 +1017,22 @@ const styles = {
         alignItems: 'center',
         padding: 20,
     },
+
+    subHeader: {
+        color: themeColors.text,
+        fontWeight: fontWeightBold,
+        fontFamily: fontFamily.GTA_BOLD,
+        fontSize: 12,
```

Comment:
Please use one of our font size variables, perhaps `variables.fontSizeSmall`

Reply:
To do (for later if you wish): we should make a global style for this kind of subheader that is just font size/weight/color. If it needs padding around it, add the padding in via a utility class.